### PR TITLE
LRQA-73526 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LRQA-73526

### CODE REVIEW

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:bundle

**Comment by Sharryxue** [comment]
Just starting review.

**Comment by melodywu** [comment]
ci:test:bundle

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/cpusersandorganizations/Organizations.testcase [hunk 1]
```diff
@@ -874,6 +878,10 @@ definition {
 		}
 
 		task ("Then the user should be created and associated under the organization") {
+			AssertTextEquals(
```
Comment (Sharryxue) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/cpusersandorganizations/Organizations.testcase]: Add the step to only one test is enough for adding the coverage. And please try to find the exist macro to use. Thanks @melodywu.

**Review by Sharryxue — CHANGES_REQUESTED** [review]
Hello @melodywu,
Add the coverage in one test is enough, find and use the exist macro please.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/cpusersandorganizations/Organizations.testcase (hunk 1 above)
Comment (melodywu) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/cpusersandorganizations/Organizations.testcase]: ok

**Pull request closed without merging**